030-Concepts/020-how-it-works.mdx

```diff
@@ -235,7 +235,7 @@ In usage, using the Xata TypeScript SDK is similar to using a Typescript ORM, fo
 
 Under the hood, however, the Xata SDK is implemented a bit differently and makes use of recent TypeScript additions to generate the types at run-time. It also has a somewhat simpler problem to solve, because it only needs to talk with the Xata API, not generate SQL. The result of this is that the Xata SDK is very lightweight and has zero dependencies, so it can be used in many JavaScript runtimes including Node.js, Cloudflare Workers, Deno, Electron, etc.
 
-## Edge caching and edge functions (Private Beta)
+## Edge caching (Beta)
```

Comment:
Let's remove caching too, it's perpetual beta and when we implement the final version it might look different